misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/LineButton.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { PureComponent } from 'react';
```

Comment:
consider moving these newly refactored files like LineButton, LineScreen, StepButton, StepScreen 
to `src/containers/activities/common/` folder
because those are no longer related to picking.